contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -87,6 +89,109 @@ var BaseCollection = Backbone.Collection.extend({
     }
 });
 
+var PageableCollection = require("backbone.paginator");
+var BasePageableCollection = PageableCollection.extend({
```

Comment:
A lot of the stuff in `BasePageableCollection` is just configuration for backbone.paginator to work with our API.